cms/envs/common.py

```diff
@@ -127,9 +127,6 @@
 
 # pylint: enable=useless-suppression
 
-################ Enable credit eligibility feature ####################
-ENABLE_CREDIT_ELIGIBILITY = True
-
```

Comment:
@feanil has a pending comment about leaving this as-is.  This is a reminder to submit that review.

Reply:
:+1: